<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addCarWindow</class>
 <widget class="QDialog" name="addCarWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>373</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Marca</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputMarca"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Modelo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputModelo"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Ano</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputAno"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Cilindrada</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputCC"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Potência</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputCV"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Matricula</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputMatricula"/>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Preço de Aluguer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inputPreco"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>85</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="goBack">
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="gravarCarro">
      <property name="text">
       <string>Gravar Carro</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
